app/client/components/identity/EmailAndMarketing/OptOutSection.tsx

```diff
@@ -12,30 +12,34 @@ interface OptOutSectionProps {
   clickHandler: ClickHandler;
 }
 
-const optOutFinder = (
-  consents: ConsentOption[],
-  clickHandler: ClickHandler
-) => (id: string) => {
-  const consent = consents.find(c => c.id === id);
-  return (
-    consent && (
-      <MarketingPreference
-        id={consent.id}
-        description={consent.description}
-        selected={consent.subscribed}
-        onClick={clickHandler}
-      />
-    )
-  );
-};
+/**
+ * NOTE:
+ * Only use this method for an OPT OUT consent, eg. "post_optout"
+ * The description of Opt On consents have changed so for UX/UI purposes they are now opt INs
+ * The backend model remains an opt OUT, so we invert the consented/subscribed value here.
+ */
+const optOutFinderAndInverter =
+  (consents: ConsentOption[], clickHandler: ClickHandler) => (id: string) => {
+    const consent = consents.find((c) => c.id === id);
+    return (
+      consent && (
+        <MarketingToggle
+          id={consent.id}
+          description={consent.description}
+          selected={!consent.subscribed} // NOTE: Opt Out consent value is inverted
```

Comment:
This is the simplest solution I found. Keeps other components generic and places the logic next to the hard coded opt outs. Tests have been added to protect the logic.